pkg/oc/cli/idle/idle.go

```diff
@@ -545,6 +560,16 @@ type scaleInfo struct {
 // scalable resources to zero, and annotating the associated endpoints objects with the scalable resources to unidle
 // when they receive traffic.
 func (o *IdleOptions) RunIdle() error {
+	clusterNetwork, err := o.OperatorClient.OperatorV1().Networks().Get("cluster", metav1.GetOptions{})
+
+	if !kerrors.IsNotFound(err) {
```

Comment:
Sorry, when I mentioned this before I hadn't looked at/considered the rest of the code.

In this case you're treating any error other than "not found" as "no error", which means you end up looking at `sdnType` even when it's uninitialized, which ends up having the right result but is still probably wrong semantically.

Probably you should just do `if err == nil` here; no error should occur, but if it does, we shouldn't fail, because this is just a helpful warning anyway.

(Also kill the blank line before the error check)